MediaRecorder.html

```diff
@@ -712,12 +713,19 @@
         <h2>Attributes</h2>
         <dl data-link-for="BlobEvent" data-dfn-for="BlobEvent" class=
         "attributes">
-          <dt><dfn><code>data</code></dfn> of type <span class=
-          "idlAttrType"><a>Blob</a></span>, readonly</dt>
-          <dd>Returns a <a
-          href="https://www.w3.org/TR/FileAPI/#blob"><code>Blob</code></a>
-          object whose type attribute indicates the encoding of the blob
-          data.</dd>
+          <dt><dfn><code>data</code></dfn> of type <span class="idlAttrType">
+          <a href="https://www.w3.org/TR/FileAPI/#blob">Blob</a></span>,
+          readonly</dt>
+          <dd>The encoded blob. Its
+          <a href="https://www.w3.org/TR/FileAPI/#dfn-type">
+          type</a> attribute correlates to the selected
+          <a href="#dom-mediarecorder-mimeType">mimeType</a>.</dd>
+
+          <dt><dfn><code>timecode</code></dfn> of type <span class "idlAttrType">
+          <a href="https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">
+          DOMHighResTimeStamp</a></span>, readonly</dt>
+          <dd><code>Timecode</code> associated with the Blob data, relative to
+          the first produced <code>blob</code>.</dd>
```

Comment:
Could you elaborate on why a `DOMHighResTimeStamp` is needed (not saying it isn't, I'd just like to know what the rationale is).

Its [definition](https://www.w3.org/TR/hr-time-2/#idl-def-domhighrestimestamp) says this value is usually a stamp relative to a well-defined [time origin](https://www.w3.org/TR/hr-time-2/#time-origin) (not true here from what I've read), OR a delta between two `DOMHighResTimeStamp`s. If we're going to use this we should at least point out that it's the latter.

Also, why call it timecode and not timestamp? Is there a good reason or precedent?